iris_grib/__init__.py

```diff
@@ -50,7 +50,7 @@
 __version__ = '0.9.0'
 
 __all__ = ['load_cubes', 'save_grib2', 'load_pairs_from_fields',
-           'save_pairs_from_cube', 'save_messages', 'GribWrapper']
```

Comment:
> Do we know why GribWrapper was in the public API but GribDataProxy wasn't?

Not really, but the original had got a bit untidy.
Before the days of the 'new strict loader', GribWrapper was what callbacks had to deal with, but now it is replaced by GribMessage in that context.

The only place a GribWrapper still rears its ugly head is when loading GRIB1 messages, which we continue to support because some standard datastreams still use them (actually, mixed GRIB1 and GRIB2 -- yuck!).
The user may possibly still care because a GribWrapper will then be passed into a load callback.
See : #56


> Further, if we've removed it from the public API do we want to mark it as "private" with a leading underscore (_) on the class name?

I don't think this is necessary, as the ```__all__ ``` should be an accepted mechanism of defining what is public.
According to [this section of PEP8](https://www.python.org/dev/peps/pep-0008/#public-and-internal-interfaces).
( which is also now directly referenced by https://github.com/SciTools/iris/blob/master/docs/iris/src/whitepapers/change_management.rst )